dts/bindings/pwm/ti,mspm0-pwm.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+#
+# SPDX-License-Identifier: LicenseRef-BnOProprietary
+
+description: |
+  Driver to enable MSPM0 PWM hardware for TIMx timers (TIMA/TIMG).
+  This driver is implementing the existing Zephyr PWM API (pwm_driver_api) and
+  supports the set_cycles and get_cycles_per_sec functions.
+  Note, supported capture channel direction for now is up to Capture Channel 2 or 4
+  (depending on the timer used) and output only.
+
+compatible: "ti,mspm0-pwm"
+
+include: [base.yaml, pinctrl-device.yaml, pwm-controller.yaml]
+
+properties:
+  ti,prescaler:
+    type: int
+    default: 0
+    description: Specifies the value used to divide the input clock frequency, allowing adjustment of the effective clock rate used by the timer.
+
+  ti,mode:
+    type: int
+    default: 0
```

Comment:
Strictly speaking, whenever you have a default value, your description _must_ explain the default. A lot of times a simple `the default value of 0 corresponds to the register reset value` is enough - but on occasion you may have some reason for choosing a specific value.

Reply:
Removed the default value and made the property required